### PR TITLE
gsim_glim_documentation 

### PR DESCRIPTION
**Pull request by rrpawar96**
## Description
GSIM_GLIM documentation

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Commit message starts with the issue number from https://issues.apache.org/jira/projects/FINERACT/. Ex: FINERACT-646 Pockets API.

- [ ] Coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions have been followed.

- [ ] API documentation at fineract-provider/src/main/resources/static/api-docs/apiLive.htm has been updated with details of any API changes.

- [ ] Integration tests have been created/updated for verifying the changes made.

- [ ] All Integrations tests are passing with the new commits.

- [ ] Submission is not a "code dump".  (Large changes can be made "in repository" via a branch.  Ask on the list.)

Our guidelines for code reviews is at https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide

### CODE REVIEW

File: fineract-provider/src/main/resources/static/api-docs/apiLive.htm [hunk 1]
```diff
@@ -9438,170 +9585,1409 @@ <h4>Retrieve Group accounts overview</h4>
...
+
+                </div>
+                <div class="method-example">
+                    <code class="method-declaration">
+GET  https://DomainName/fineract-provider/api/v1/groups/{groupId}/gsimaccounts
+                    </code>
+
+
+                    <code class="method-response">
+[{
```
Comment (nikpawar89) [inline fineract-provider/src/main/resources/static/api-docs/apiLive.htm:9770]: check the formatting of other apis and align it accordingly. 
you may also use this tool for formatting the JSON body:
https://jsonformatter.curiousconcept.com/

**Review by nikpawar89 — CHANGES_REQUESTED** [review]
Please format the JSON body.  @rrpawar96  if documentation was tested, then this PR has go ahead from me.  You can use tools like https://jsonformatter.curiousconcept.com/ to format JSON body.

**Comment by vorburger** [comment]
@nikpawar89 OK for you if we merge this now? Just double checking, given your previous review (above).

**Comment by vorburger** [comment]
FTR #943 will not reformat this `fineract-provider/src/main/resources/static/api-docs/apiLive.htm` so we don't have to wait from either one on the other to avoid merge conflicts... I'll go ahead with #943 later today, and this can be separately merged any time later.

**Comment by nikpawar89** [comment]
LGM,  @vorburger whenever you have time please merge this PR.  Thanks!
@rrpawar96  good work!

**Pull request merged**